rules/BUILD

```diff
@@ -13,9 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-# TODO(aiuto): Add dependency on Skylib for bzl_library so that we can later
-# generate docs with stardoc.
+"""Rules for declaring licenses used."""
```

Comment:
General comment—

Running `buildifier --lint=warn --warnings=all -v -r .`  on this branch produced 25 Buildifier warnings. I created a PR into the `aiuto:linty` branch fixing all 25 warnings: **https://github.com/aiuto/rules_license/pull/1**

You can merge my PR into this branch if you want the changes or ignore the PR if you'd like to put off fixing these Buildifier warnings.

Reply:
Gaggggg...    Sometimes I think buildifier is strict without need.
https://github.com/bazelbuild/buildtools/issues/1035

The sad truth is that Google's internal buildifier does not complain about these things, probably because the users would not tolerate massive rewrites on nits.

I'll make the changes anyway for now, because winning a fight against tools is low on my priorities.